amr-wind/physics/SyntheticTurbulence.cpp

```diff
@@ -268,8 +268,8 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE void get_lr_indices(
     }
 
     const amrex::Real xfrac = xbox - turb_grid.dx[dir] * il;
-    rxl = xfrac / turb_grid.dx[dir];
-    rxr = (1.0 - rxl);
+    rxr = xfrac / turb_grid.dx[dir];
+    rxl = (1.0 - rxr);
```

Comment:
Hmm... are we sure about this?

Reply:
If `xfrac` is zero, the weight of the right point should be zero and the weight of the left point should be 1, right? This was a sample of some distributions on a grid finer than the turbulence grid before the fix:
![pre_fix_distributions](https://user-images.githubusercontent.com/26737762/190210570-e2ffe972-51c8-4cc5-b2f5-1cbc62623739.png)
And this is after the fix:
![post_fix_distributions](https://user-images.githubusercontent.com/26737762/190211197-45f49e37-6b12-4ca1-bc55-fe1b710a59f7.png)